resources/scripts.js

```diff
@@ -158,21 +178,25 @@ function chipChange(loc, color) {
     return;
 }
 
-function playerWin() {
+function playerWin(type) {
     displayMessage('You win!');
+    if (type === 'come') {
```

Comment:
Try using constants instead of a straight string check

Reply:
Created constants for all strings